lib/fleiss/backend/active_record/concern.rb

```diff
@@ -14,6 +16,15 @@ module Concern
           scope :started,      -> { where(arel_table[:started_at].not_eq(nil)) }
           scope :not_started,  -> { where(arel_table[:started_at].eq(nil)) }
           scope :scheduled,    ->(now = Time.zone.now) { where(arel_table[:scheduled_at].gt(now)) }
+
+          scope :lock_expired, lambda {|now = Time.zone.now, lock_ttl = DEFAULT_LOCK_TTL|
+            if lock_ttl
+              min_started_at = now - (1.1 * lock_ttl) # 10% threshold
+              where(finished_at: nil).where(arel_table[:started_at].lt(min_started_at)) # not yet finished and started at quite a while ago
+            else
+              none # do nothing unless explicitly configured
+            end
```

Comment:
No expires_at check is needed here - this method is composed with other scopes down the code, in `pending` class method.